src/terraform-resource/terraform/planfile.go

```diff
@@ -0,0 +1,78 @@
+package terraform
```

Comment:
There's a bunch of duplication between this file and `StateFile`. Could probably extract the common functionality into a different struct, e.g. `RemoteFile`.